digdag-tests/src/test/java/acceptance/ParamSetRedisIT.java

```diff
@@ -36,6 +40,13 @@ public void setValueToRedis()
                 projectDir.resolve("set.dig").toString()
         );
         assertCommandStatus(status);
+
+        long key1_ttl = redisClient.ttl("0:key1");
+        long key2_ttl = redisClient.ttl("0:key2");
+
+        assertTrue(key1_ttl < TTL_90_DAYS_PLUS_1SEC && key1_ttl > TTL_89_DAYS);
+        assertTrue(key2_ttl < TTL_90_DAYS_PLUS_1SEC && key2_ttl > TTL_89_DAYS);
+
```

Comment:
Basic question. Why compare as ttl < TTL_90_DAYS_PLUS_1SEC not ttl <= TTL_90_DAYS?

Reply:
To tell the truth, I have no idea how to test `param_set>` with the Redis server.

`param_set>`  set parameter X with TTL 90days.(60sec * 60min * 24 hours * 90days)
Redis can't set past TTL like PostgreSQL.

If assert check immediately after `param_set`, It will be `key1_ttl == TTL_90_DAYS`.
But if that test delay some reason, The TTL will `TTL_90_DAYS - XXsec`
That's why I checked >= TTL_89_DAYS

If TTL configurable, that test will be easy.

* set TTL 1sec;
* `param_set>` xxx
* wait 1sec
* Check the value already expire.

Reply:
Thanks!